test/library/standard/Path/dlongnecke/normPath/normPath.preexec

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# Python (2.7) code with proper output for os.path.normpath(). Use this to
+# generate the output for "testcases_normPath.good".
+
+# NOTE: I had to revert this script back to Python 2.7 because a remote
+# testing machine did not support Python 3. It is assumed that the
+# implementation of os.path.normpath() has not changed much between Python 2.7
+# and Python 3.7 (I referenced the latter for my implementation).
+
+
+import os.path
+import sys
+
+
+# As per the description of normPath.
+collapsedA = 'foo/bar'
+collapseToA0 = 'foo//bar'
+collapseToA1 = 'foo/bar/'
+collapseToA2 = 'foo/./bar'
+collapseToA3 = 'foo/baz/../bar'
+
+# Expanded test coverage.
+multiCurDir = '././.'
+multiParentDir = '../../..'
+multiMixedParentCurDir = './.././.././..'
+multiOnlyPathSep = '/////'
+curDir = '.'
+parentDir = '..'
+pathSep = '/'
+pathSep2 = '//'
+
+# Additional tests for Cygwin.
+onCygwin0 = 'foo\bar/baz'
```

Comment:
These guys don't seem to be used in the file, might be good to comment them out and/or indicate why that is for posterity